src/js/components/Diagram/Diagram.js

```diff
@@ -215,6 +225,7 @@ const Diagram = forwardRef(({ connections, ...rest }, ref) => {
             <path
               // eslint-disable-next-line react/no-array-index-key
               key={index}
+              animation={animation || undefined}
```

Comment:
wouldn't `animation` be undefined as the default? If so, this should work.
```suggestion
              animation={animation}
```

Reply:
Tested this out locally and everything looks good.